tests/NLog.UnitTests/Targets/FileTargetTests.cs

```diff
@@ -2770,8 +2837,8 @@ public void MaxArchiveFilesWithDate_in_sameDir(bool changeCreationAndWriteTime)
                     var time = now.AddDays(i);
                     if (changeCreationAndWriteTime)
                     {
-                        File.SetCreationTime(filePath, time);
-                        File.SetLastWriteTime(filePath, time);
+                        File.SetCreationTimeUtc(filePath, time);
```

Comment:
Leftovers from when I was debugging the failing unit-tests on Mono/Travis. Will undo the changes.